rmgpy/rmg/react.py

```diff
@@ -57,14 +59,21 @@ def react(*spcTuples):
     """
 
     from rmgpy.rmg.main import maxproc
-    
-    # Available RAM memory (kB)
-    mem = psutil.virtual_memory()
-    usermem = mem.available
-    
-    tmp = divmod(usermem, resource.getrusage(resource.RUSAGE_SELF).ru_maxrss)
+   
+    # Get available RAM (GB)and procnum dependent on OS
+    if platform == "linux" or platform == "linux2":
```

Comment:
You could use `platform.startswith('linux')` here to check for both.

Reply:
Thanks! Done.